server_functions/auth.js

```
@@ -522,314 +566,286 @@ async function createNewClient(
+  if (!cookies().has(SESSION_COOKIE_NAME)) {
+    return null;
+  }
+  try {
+    const sessionCookie = JSON.parse(cookies().get(SESSION_COOKIE_NAME).value);
+    const { account } = await createSessionClient(sessionCookie.session);
+    return account.get();
+  } catch (error) {
     return null;
+  }
+
+  return null;
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement, ...

... this statement, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)